beacon_node/lighthouse_network/src/rpc/codec/ssz_snappy.rs

```diff
@@ -625,14 +624,18 @@ fn handle_rpc_response<E: EthSpec>(
             )),
         },
         SupportedProtocol::DataColumnsByRootV1 => match fork_name {
-            // TODO(das): update fork name
-            Some(ForkName::Deneb) => Ok(Some(RPCResponse::DataColumnsByRoot(Arc::new(
-                DataColumnSidecar::from_ssz_bytes(decoded_buffer)?,
-            )))),
-            Some(_) => Err(RPCError::ErrorResponse(
-                RPCResponseErrorCode::InvalidRequest,
-                "Invalid fork name for data columns by root".to_string(),
-            )),
+            Some(fork_name) => {
+                if fork_name.deneb_enabled() {
```

Comment:
we want `electra_enabled` here right? otherwise we'll be incorrectly advertising this protocol as supported immediately on mainnet when we release

Reply:
Good catch! we can perhaps add a `peer_das_enabled` function on `chain_spec` so we can test this on deneb-based devnets.

`if self.fork_context.spec.peer_das_enabled()`

Reply:
PeerDAS is currently supported for both deneb and electra. This check does not advertise the topic on deneb, simply allows it to decode it. Advertise logic is in `SupportedTopic::currently_supported`.

Reply:
> This check does not advertise the topic on deneb, simply allows it to decode it. Advertise logic is in SupportedTopic::currently_supported.

Ah right, thanks